app/src/main/res/layout/activity_itinerary_show.xml

```diff
@@ -12,6 +12,22 @@
     tools:context=".ui.mission.ItineraryShowActivity">
 
 
+    <com.google.android.material.floatingactionbutton.FloatingActionButton
+        android:id="@+id/editButton"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_gravity="bottom|end"
```

Comment:
I think that you don't have to use gravity in a constraintlayout if you place your constraints correctly, but it's a detail